pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/AdminResource.java

```diff
@@ -254,6 +254,15 @@ protected void validateTopicName(String property, String namespace, String encod
         this.topicName = TopicName.get(domain(), namespaceName, topic);
     }
 
+    protected void validatePartitionedTopicName(String tenant, String namespace, String encodedTopic) {
+        // first, it has to be a validate topic name
+        validateTopicName(tenant, namespace, encodedTopic);
+        // second, "-partition-" is not allowed
+        if (encodedTopic.contains("-partition-")) {
```

Comment:
can we use `TopicName.PARTITIONED_TOPIC_SUFFIX` instead a plain string?

Reply:
changed it to use the constant